examples/search.rs

```diff
@@ -1,17 +1,17 @@
-use anyhow::Result;
-use qdrant_client::prelude::*;
+use qdrant_client::payload::Payload;
 use qdrant_client::qdrant::{
-    Condition, CreateCollectionBuilder, Distance, Filter, PayloadIncludeSelector, QuantizationType,
-    ScalarQuantizationBuilder, SearchParamsBuilder, SearchPointsBuilder, VectorParamsBuilder,
+    Condition, CreateCollectionBuilder, Distance, Filter, PointStruct, QuantizationType,
+    ScalarQuantizationBuilder, SearchParamsBuilder, SearchPointsBuilder, UpsertPointsBuilder,
+    VectorParamsBuilder,
 };
-use serde_json::json;
+use qdrant_client::{Qdrant, QdrantConfig, Result};
 
 #[tokio::main]
 async fn main() -> Result<()> {
     // Example of top level client
     // You may also use tonic-generated client from `src/qdrant.rs`
-    let config = QdrantClientConfig::from_url("http://localhost:6334");
-    let client = QdrantClient::new(Some(config))?;
+    let config = QdrantConfig::from_url("http://localhost:6334");
+    let client = Qdrant::new(Some(config))?;
```

Comment:
```suggestion
    let client = Qdrant::from_url("http://localhost:6334").build()?;
```

Reply:
Tackled in 74ef1441bde64143164d0e5189d6b100f3178144